apps/nerves_hub_web_core/lib/nerves_hub_web_core/devices.ex

```diff
@@ -327,14 +327,15 @@ defmodule NervesHubWebCore.Devices do
   def get_ca_certificates(%Org{id: org_id}) do
     from(ca in CACertificate, where: ca.org_id == ^org_id)
     |> Repo.all()
+    |> Repo.preload([:jitp])
```

Comment:
```suggestion
    from(ca in CACertificate, where: ca.org_id == ^org_id, preload: :jitp)
    |> Repo.all()
```
`Repo.preload` called after the query will loop through the list and do more lookups to preload. Its prob better off to include the lookups as part of the query transaction